src/core/settings/qgssettings.h

```diff
@@ -103,6 +103,7 @@ class CORE_EXPORT QgsSettings : public QObject
         static const inline char *QGIS_DIGITIZING_SHAPEMAPTOOLS = "qgis/digitizing/shape-map-tools";
         static const inline char *QGIS_NETWORKANDPROXY = "qgis/networkAndProxy";
         static const inline char *SVG = "svg";
+        static const inline char *LAYER_TREE = "core/layer_tree";
```

Comment:
```suggestion
        static const inline char *CORE_LAYERTREE = "core/layer-tree";
```
my idea was to use kebab case for the string, and for the var name, keep underscores for levels distinction.
See QGIS_DIGITIZING_SHAPEMAPTOOLS 3 lines above.
But I'm not a super fan for the var names. Do you think LAYER_TREE only is better?

Reply:
> my idea was to use kebab case for the string, and for the var name, keep underscores for levels distinction.

Ok, sounds good

>  Do you think LAYER_TREE only is better?

I'm not fussed either way. It's hidden from sip anyway so we can easily change later if we want.